tests/ops/util/test_tc_string.py

```diff
@@ -335,7 +335,7 @@ def test_filter_invalid_vendor_legal_basis(self):
         m = TCModel(
             vendor_legitimate_interests=[66]
         )  # This vendor doesn't have leg int purposes, or special purposes. It does have flexible purposes,
-        # but this isn't an option right now, as is_service_specific is False
+        # but this isn't an option right now, as is_service_specific is True
```

Comment:
hm what does the `is_service_specific` value have to do with flexible purposes?

Reply:
thank you for catching my find and replace mistake!

Reply:
This does highlight how the backend isn't accounting for flexible purposes when building the TC string though!

Reply:
I don't have any publisher restrictions-type handling for TC strings on the backend: https://github.com/ethyca/fides/blob/main/src/fides/api/util/tcf/tc_model.py#L297, do you have it on the FE?

Reply:
nope not right now, I haven't really thought about it...

Reply:
sounds good at least we match and we can add later as needed!